etc/sources.csv

```diff
@@ -1,22 +1,22 @@
 author,year,matchauthoryear,matchtitle,matchtype,matchid
 Shiraliev,1997,shiraliev1997,Azerbajdzhanskij jazyk,,wals:5521
 Mehraliev,p.c. 2017,,,,
-Shiraliev,1971,,,,
+Shiraliev,1971,,,,hh:g:Shiraliev:Azerbajdzanskogo
 Poppe,1964,nicholas1964,Bashkir Manual,grammar,hh:g:Poppe:Bashkir
 Gisbert Fanselow,p.c. 2017,,,,
-Yuldashev,1981,,,,
-Yuldashev,1997,,,,
-Garinov,1959,,,,
+Yuldashev,1981,,,,mpieva:Juldasev1981Grammati
+Yuldashev,1997,,,,wals:3027
+Garipov,1959,,,,mpieva:Garipov1959Baskirsk
 Dmitriev,1948,dmitriev1948,Grammatika bashkirskogo jazyka,grammar,hh:g:Dmitriev:Bashkirskogo
 Juwon,2011,juwon2011,A grammar of Ewen,grammar,hh:g:Juwon:Ewen
-Pakendorf,in preparation,,,,
+Pakendorf,2020,,The North Siberian Turkic Languages,,
+Pakendorf,2020,,Even and the Northern Tungusic Languages,,

```

Comment:
This will require updating the references in the data, too, I guess. If the sources don't overlap for the same languages, we could do that automatically.

Reply:
I have updated the references in the data accordingly. This book is not on Glottolog yet and there are two chapters with the same author and the same year, but referring to different languages, how do we link it?